documentation/asciidoc/topics/ref_server_configuration.adoc

```diff
@@ -0,0 +1,27 @@
+//community and downstream
+[id='server-configuration_{context}']
+= {brandname} Server configuration
+
+[role="_abstract"]
+{brandname} Server configuration examples.
+
+`deploy.infinispan`:: Configures cache managers and {brandname} Server endpoints, interfaces, socket bindings, and security.
+
+[discrete]
+== Cache Manager configuration values
+[source,options="nowrap",subs=attributes+]
+----
+include::yaml/cache-manager-configuration.yaml[]
+----
+
+[discrete]
+== Endpoint configuration values
+[source,options="nowrap",subs=attributes+]
+----
+include::yaml/endpoint-configuration.yaml[]
+----
+
```

Comment:
Another thing we should do here is move this paragraph from the other table here: https://github.com/infinispan/infinispan-helm-charts/blob/main/documentation/asciidoc/topics/ref_deployment_configuration_values.adoc#L35

"You should not change the default socket bindings or the security realm and endpoints named "metrics". Modifying these default properties can result in unexpected behavior and loss of service."

```
[IMPORTANT]
====
To avoid unexpected behavior and loss of service you should not change some default configuration. 
Do not modify the following values:

* Socket bindings named "default" or "metrics". 
* Security realm named "metrics".
* Endpoint named "metrics".
====
```

Not sure how that sounds but something along those lines might be good.

Reply:
I added the admonition. This is important info. 
+1 to adding other configuration sections. The questions is whether we'll add some custom configuration or keep the default. If default then I'm inclined to keep it simple and add comments in the values.yaml (as @ryanemerson suggested).